BUILD

```diff
@@ -46,6 +46,22 @@ cc_binary(
     ]),
 )
 
+cc_binary(
+    name = "_XLAC_cuda_functions.so",
+    copts = [
+        "-fopenmp",
```

Comment:
I wonder how the copts and linkopts are determined

Reply:
To be honest, I just copied them from `_XLAC`. I guess I could get rid of them, though. What do you think?

Reply:
Well, if it works now, feel free to keep it :p